src/DetailsView/details-view-initializer.ts

```diff
@@ -449,6 +455,19 @@ if (isNaN(tabId) === false) {
     );
 }
 
-function createNullifiedRenderer(doc, render): DetailsViewRenderer {
-    return new DetailsViewRenderer(null, doc, render, documentElementSetter);
+function createNullifiedRenderer(
+    doc: Document,
+    render: typeof ReactDOM.render,
+): NoContentAvailableViewRenderer {
+    // using an instance of an actual store (instead of a StoreProxy) so we can get the default state.
+    const store = new UserConfigurationStore(null, new UserConfigurationActions(), null);
```

Comment:
would be nice to not have to do this ... but I don't really have a better suggestion.

Reply:
I think we can improve this... but I try to balance complexity against having the fix as soon as possible. We'll get there.